app/controllers/catalog_controller.rb

```diff
@@ -114,8 +114,9 @@ def self.chf_search_fields
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field solr_name("title", :stored_searchable), label: "Title", itemprop: 'name', if: false
+    config.add_index_field solr_name("additional_title", :stored_searchable), label: "Additional Title", itemprop: 'name', if: :present?
```

Comment:
I honestly don't really know what itemprop is. I'm not sure it's used for much of anything. I think it's supposed to be used for schema.org metadata markup generation (that's where I recognize the term 'itemprop' from), but does not seem to be doing this in our app. It's kind of another mystery of the hydra stack to me at the moment.